src/DB/postgres.ts

```diff
@@ -0,0 +1,19 @@
+import dotenv from 'dotenv';
+dotenv.config();
+import { Client, Pool } from 'pg'
+
+
+const pool = new Pool({
+  host: process.env.POSTGRES_HOST,
+  user: process.env.POSTGRES_USER,
+  password: process.env.POSTGRES_PASSWORD,
+  database: process.env.POSTGRES_DATABASE,
+})
+ 
+export const query = async (text: string, params: any) => {
```

Comment:
'any' is best avoided, can we replace it with a determined type?

Reply:
This came direct from the suggested architecture by the  node-postgres library that I had picked. It can probably be typed but I think it's generally dynamic based on the query string being passed in.